cubano-core/src/main/java/org/concordion/cubano/driver/action/ActionWait.java

```diff
@@ -94,17 +95,38 @@
     }
 
     /**
-     * Sets how long to wait for the evaluated condition to be true. Required.
+     * Sets how long to wait for the evaluated condition to be true.
+     * Either MaxAttempts or Timeout must be set, but they cannot be used together.
```

Comment:
It would be better to make ActionWait abstract and create subclasses - eg. TimedActionWait and CountedActionWait (I think these names can be improved)

Reply:
Hmm, a good idea.  I'll think on some names...